setup.py

```diff
@@ -13,8 +23,68 @@ def get_tag(self):
         return python, abi, plat
 
 
+def download_so():
+    uname = platform.uname()
+    machine = uname.machine
+
+    # do not download if target platfrom dll not found
```

Comment:
move this comment to proper place.

Reply:
Feel free to delete it in your future PR.